pkg/kademlia/lookup.go

```diff
@@ -5,24 +5,30 @@ import (
 	"context"
 	"log"
 	"math/big"
+	"sync"
 	"time"
 
 	"storj.io/storj/pkg/dht"
 	"storj.io/storj/pkg/node"
 	"storj.io/storj/pkg/pb"
 )
 
-type sequentialLookup struct {
+var (
+	defaultRoutines = 5
+)
+
+type lookup struct {
```

Comment:
let's rename the type to `discovery` to make it more distinguished from other operations